### ISSUE TITLE
Krisp/Noise Supression Toggle Keybind

### ISSUE THREAD

**Issue by TrueReplayJay** [Vencord/plugin-requests#156]
### Plugin Description

Though specific, I believe having the ability to set a key bind to toggle noise suppression would be greatly helpful. If I would like to use an external soundboard or play a sound in a call, it is difficult to do so without tabbing into discord and fiddling around in the settings. I have heard other's have the same complaint. A plugin could relatively easily fix this problem.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Vendicated** [comment]
way too specific / not useful

plus you dont have to go into settings
[Screencast from 2024-03-05 22-31-02.webm](https://github.com/Vencord/plugin-requests/assets/45497981/510c5ced-4a9c-4afe-9fbd-20de0ee1cc6f)

**Comment by TrueReplayJay** [comment]
> way too specific / not useful

This is true, but this could be just one feature in a larger "key binds" plugin, filling in some of the gaps in discord's own key binds. If we can have a plugin to change the ban gif I feel like this should be somewhere on the acceptable usefulness scale lol.
